presto-cache/src/main/java/com/facebook/presto/cache/CacheStats.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cache;
+
+import org.weakref.jmx.Managed;
+
+import javax.annotation.concurrent.ThreadSafe;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+@ThreadSafe
+public class CacheStats
```

Comment:
Do we really need a new one =)?

Reply:
Yes! loading cache `LocalRangeCacheManager` doesn't work. @jessesleeping and I try to figure out if there is way out but we failed to do so. We need our own control on cache